dex/src/main/resources/application.conf

```diff
@@ -786,10 +786,101 @@ kanela {
     show-banner = false
 }
 
-# Performance metrics
 kamon {
-  # Set to "yes", if you want to report metrics
-  enable = no
```

Comment:
Could you return the previous code here? ) Kamon should be disabled by default.

Reply:
fixed